src/tasks/tasks.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { Controller, Post, Body } from "@nestjs/common";
+
+import { TasksService } from "./tasks.service";
+import { Task } from "../common/entities/task.entity";
+
+@Controller("tasks")
+export class TasksController {
+  constructor(private readonly tasksService: TasksService) {}
+
+  @Post()
+  async createTask(@Body() taskData: Partial<Task>): Promise<Task> {
```

Comment:
```suggestion
  async createTask(@Body() taskData: CreateTaskDto): Promise<Task> {
```
Defining the type to be partial makes all fields optional but we most of the fields are mandatory for the create action. You need to create a dto (data transfer object) class and pass it here.

You should create another file tasks.dtos.ts and pass the dtos from there. Look into [class-validator](https://docs.nestjs.com/techniques/validation) for help with creating the dto.